README.md

```diff
@@ -126,11 +125,10 @@ Spinnaker. It is likely different from the operator's namespace.
 
 ### Example 1: Basic Install
 
-In `deploy/spinnaker/basic/spinnakerservice.yml`, change the `config.persistentStorage` section to point to an S3 bucket you own or use a different persistent storage. Also make sure to update the Spinnaker version to the [desired version](https://www.spinnaker.io/community/releases/versions/#latest-stable).
+In `deploy/spinnaker/basic/spinnakerservice.yml`, change the `config.persistentStorage` section to point to an S3 bucket you own or use a different persistent storage. Also make sure to update the Spinnaker version to the [desired version](https://www.spinnaker.io/community/releases/versions/#latest-stable). Since you installed Operator in `basic` mode, you must use the `spinnaker-operator` namespace. The permissions in `basic` mode are scoped to a single namespace so it doesn't see anything in other namespaces.
```

Comment:
> you must use the `spinnaker-operator` namespace
There's no technical reason for that. In basic mode, you can specify the namespace on the command line (`kubectl create ns blah && kubectl -n blah apply -f deploy/operator/basic && kubectl -n blah apply -f deploy/spinnaker/basic/spinnakerservice.yml`). In cluster mode, the namespace is also in the role binding, that's why we strongly suggest `spinnaker-operator` as a default.